External/FEXCore/include/FEXCore/Debug/InternalThreadState.h

```diff
@@ -62,6 +63,14 @@ namespace FEXCore::Core {
     fextl::vector<DebugDataSubblock> Subblocks;
     fextl::vector<DebugDataGuestOpcode> GuestOpcodes;
     fextl::vector<FEXCore::CPU::Relocation> *Relocations;
+
+    void *operator new(size_t size) {
```

Comment:
More leftovers (?)

Reply:
Required due to raw new usage. Added a comment.